app/controllers/users.js

```diff
@@ -0,0 +1,15 @@
+const util = require('util');
+const logger = require('../logger');
+const usersService = require('../services/users');
+
+exports.signUp = (req, res, next) =>
+  usersService
+    .signUp(req.body)
+    .then(() => {
+      logger.info(`User created: ${req.body.first_name}`);
+      return res.status(201).send({ firstName: req.body.first_name });
```

Comment:
We use input/output of params in snake_case
```suggestion
      return res.status(201).send({ first_name: req.body.first_name });
```